README.md

```diff
@@ -1,68 +1,131 @@
 - [Delving Bitcoin](https://delvingbitcoin.org/) ([cron](.github/workflows/delving-bitcoin.yml), [source](delvingbitcoin_2_elasticsearch))
 
 Weekly
+
 - [bitcoin.stackexchange](https://bitcoin.stackexchange.com/) ([cron](.github/workflows/stackexchange.yml), [source](bitcoin.stackexchange.com))
 - Bitcoin Talk Forum ([cron](.github/workflows/bitcointalk.yml), [source](bitcointalk))
     - only the [Development & Technical Discussion Board](https://bitcointalk.org/index.php?board=6.0)
     - only for specific authors
 - [Bitcoin Transcript](https://btctranscripts.com/) ([cron](.github/workflows/bitcointranscripts.yml), [source](bitcointranscripts))
 - [Bitcoin Optech](https://bitcoinops.org/) ([cron](.github/workflows/bitcoinops.yml), [source](bitcoinops))
 
-Additionally, for on-demand scraping tasks, we utilize a Scrapybot, details of which can be found in the [Scrapybot section](#scrapybot) below.
-
-## Setup
-
-You need an env file to indicate where you are pushing the data.
-
-1. Copy `cp .env.sample .env`
-2. run `cd common && yarn install && cd ../mailing-list && yarn install && cd ..`
-3. To scrape a mailing list run `node mailing-list/main.js` with additional env vars like `URL='https://lists.linuxfoundation.org/pipermail/bitcoin-dev/'` and `NAME='bitcoin'`
-3a. Or you can do something like `cd bitcoin.stackexchange.com && pip install -r requirements.txt && cd .. && python3 bitcoin.stackexchange.com/main.py`
-
-You should be calling the scrapers from the root dir because they use the common dir.
-
-## Scrapybot 
-
-We have implemented a variety of crawlers (spiders), each designed for a specific website of interest.
-You can find all the spiders in the [`scrapybot/scrapybot/spiders`](scrapybot/scrapybot/spiders) directory.
-
-This section explains how to run the scrapers in the `scrapybot` folder.
-
-To run a crawler using scrapybot, for example `rusty`, which will scrape the site `https://rusty.ozlabs.org`, switch to the root directory(where there is this README file) and run these commands from your terminal:
-- `pip install -r requirements.txt && cd scrapybot`
-- `scrapy crawl rusty -O rusty.json`
-
-The above commands will install scrapy dependencies, then run the `rusty` spider(one of the crawlers) and store the collected document in `rusty.json` file in the `scrapybot` project directory.
-
-The same procedure can be applied to any of the crawlers in the `scrapybot/spiders` directory.
-There is also a script in `scrapybot` directory called `scraper.sh` which can run all the spiders at once.
-
-
-### Sending the output to elastic search
+## Prerequisites
+
+Before you begin, ensure you have following installed:
+
+- [Python 3.8+](https://www.python.org/downloads/)
+- [Node.js 14+](https://nodejs.org/)
+- [pip](https://pip.pypa.io/en/stable/)
+- [yarn](https://classic.yarnpkg.com/en/docs/install/)
+- [Elasticsearch](https://www.elastic.co/downloads/elasticsearch)
```

Comment:
We recently moved away from nodejs implemented scrapers, right? so nodejs and yarn are probably not needed in this list. Also elasticsearch is installed as part of `requirements.txt`, so I feel that this line will create confusion for people.